### DISCUSSION TITLE
Unable to add a duplicate FQDN in provision when Overseer and FL servers specify the same FQDN.

### DISCUSSION THREAD

**Discussion by asus-ocis** [NVIDIA/NVFlare#710] [Q&A]
Hi all,

I want to deploy nvflare 2.1 on k8s. Overseer, server1 and server2 will run on different pods,
We have a HAproxy on k8s, so we need to set the same FQDN.

my project requirements:
```
Overseer : 
  fqdn : myk8s.com
  port: 7001
server1:
   fqdn: myk8s.com
   learn_port:70032
   admin_port:70033
server2:
   fqdn: myk8s.com
   learn_port:70055
   admin_port:70052   

```
But I got the following error after executing provison.
`ValueError: Unable to add a duplicate name myk8s.com into this project.`

Check spec.py
https://github.com/NVIDIA/NVFlare/blob/a23afd7759b36f3885c9de68b461c5ea8358d085/nvflare/lighter/spec.py#L72
The name field in the participants needs to be unique.

Is it possible to add FQDN field for Overseer and FL Server in project.yml in future versions? Or have any other suggestion?

**Comment by YuanTingHsieh** [comment]
Thanks for the question.

@IsaacYangSLA and @chesterxgchen what do you think about this?

**Reply by chesterxgchen** [reply]
I think we should address this in next version

**Reply by asus-ocis** [reply]
Thanks for your reply.
